### PR TITLE
feat(files): custom add to cart & add to wishlist component

### PR DESCRIPTION
**Pull request by kibo-amolpawar**

### CODE REVIEW

**Review by kibo-kevinwatts — CHANGES_REQUESTED** [review]
Make Button Text props so it can be easily customized
Rename component to "KiboProductActions"

File: components/customcomponents/CustomAddToCart.vue [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    <slot name="add-to-wishlist">
+      <div class="column">
+        <SfButton class="sf-add-to-wishlist__button" :disabled="disabled" @click="addToWishList">
+          Add to Wishlist
+        </SfButton>
+      </div>
+    </slot>
+  </div>
+</template>
+<script>
```
Comment (kibo-sushant) [inline components/customcomponents/CustomAddToCart.vue]: Put lang="ts"

File: components/customcomponents/CustomAddToCart.vue [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      default: false,
+    },
+    /**
+     * Selected quantity
+     */
+    qty: {
+      type: [Number, String],
+      default: 1,
+    },
+    quantityleft: {
```
Comment (kibo-sushant) [inline components/customcomponents/CustomAddToCart.vue]: use camelCase conventions like quantityLeft

File: components/customcomponents/CustomAddToCart.vue [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+     */
+    disabled: {
+      type: Boolean,
+      default: false,
+    },
+    /**
+     * Selected quantity
+     */
+    qty: {
+      type: [Number, String],
```
Comment (kibo-sushant) [inline components/customcomponents/CustomAddToCart.vue]: Do we need String here?

File: components/customcomponents/CustomAddToCart.vue [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+<style lang="scss" scoped>
+.sf-add-to-cart {
+  flex-wrap: wrap;
+  &__button {
+    width: 11.6rem; //186px
+  }
+}
+.sf-add-to-wishlist {
+  &__button {
+    width: 11.6rem; //186px
```
Comment (kibo-sushant) [inline components/customcomponents/CustomAddToCart.vue]: Can we avoid the repetition of "width: 11.6rem"?

File: components/customcomponents/CustomAddToCart.vue [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+.quantity-left {
+  color: var(--_c-dark-primary);
+  font-size: var(--font-size--xs);
+  font-style: italic;
+  line-height: var(--font-size--sm);
+  padding-left: 20px;
+}
+.column {
+  flex: 50%;
+  margin-bottom: 12px;
```
Comment (kibo-sushant) [inline components/customcomponents/CustomAddToCart.vue]: Instead of px either use rem or %

File: components/customcomponents/CustomAddToCart.vue [hunk 6]
```diff
@@ -0,0 +1,105 @@
+<template>
```
Comment (kibo-sushant) [inline components/customcomponents/CustomAddToCart.vue]: rename CustomAddToCart.vue to KiboAddToCart.vue

Comment (kibo-amolpawar) [reply components/customcomponents/CustomAddToCart.vue]: Renamed & Moved to components directory

File: components/customcomponents/CustomAddToCart.vue (hunk 1 above)
Comment (kibo-amolpawar) [reply components/customcomponents/CustomAddToCart.vue]: Added in script tag

File: components/customcomponents/CustomAddToCart.vue (hunk 4 above)
Comment (kibo-amolpawar) [reply components/customcomponents/CustomAddToCart.vue]: Resolved. add variable

File: components/customcomponents/CustomAddToCart.vue (hunk 5 above)
Comment (kibo-amolpawar) [reply components/customcomponents/CustomAddToCart.vue]: Resolved. converted to rem

File: components/customcomponents/CustomAddToCart.vue (hunk 2 above)
Comment (kibo-amolpawar) [reply components/customcomponents/CustomAddToCart.vue]: modified to camelCase

File: components/customcomponents/CustomAddToCart.vue (hunk 3 above)
Comment (kibo-amolpawar) [reply components/customcomponents/CustomAddToCart.vue]: Existing  SfAddToCart component have both the types so kept as it is

**Comment by kibo-amolpawar** [comment]
> Make Button Text props so it can be easily customized Rename component to "KiboProductActions"

**Review by kibo-kevinwatts — APPROVED** [review]

**Pull request merged**